policyengine_canada/variables/gov/provinces/qc/tax/income/credits/work_premium/qc_work_premium_supplement_amount.py

```diff
@@ -0,0 +1,26 @@
+from policyengine_canada.model_api import *
+
+
+class qc_work_premium_supplement_amount(Variable):
+    value_type = float
+    entity = Household
+    label = "Quebec supplement to the work premium"
+    definition_period = YEAR
+    defined_for = ProvinceCode.QC
+
+    def formula(household, period, parameters):
+        p = parameters(
+            period
+        ).gov.provinces.qc.tax.income.credits.work_premium.supplement
+
+        person = household.members
+        work_income_eligible = (
+            person("working_income", period) > p.work_income_eligibility
+        )
+        supplement_eligible = household.sum(work_income_eligible)
```

Comment:
what if there are 3 people in the household that are work income eligible (child)

Reply:
The supplement to the work premium is only for the tax filer and its spouse